recipes/glacier_lengths/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = 'glacier_lengths' %}
+{% set version = '0.0.4' %}
+
+package:
+  name: '{{ name | lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/erikmannerfelt/glacier_lengths/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 93843906e15d5bbe1b8a7e457c75d1bbe6fc5ea3ccc9fc5e3229c4f9a563d186
+
+build:
+  number: 0
+  noarch: python
+  script: '{{ PYTHON }} -m pip install . -vv'
+
+requirements:
+  host:
+  - python >=3.7
+  - pip
+  - shapely
+  - numpy
```

Comment:
You probably don't need shapely and numpy on host, unless this package imports itself at setup.py.

Reply:
The package is imported to fetch the version. Maybe it should be the other way haha.. Shapely and numpy are the only dependencies, however, so it's pretty fast, right?

Reply:
No other way unless they fix it upstream :-(